apps/safe-claiming-app/src/widgets/Widget.tsx

```diff
@@ -0,0 +1,213 @@
+import { useSafeAppsSDK } from "@gnosis.pm/safe-apps-react-sdk"
+import {
+  Box,
+  Button,
+  type ButtonProps,
+  styled,
+  Typography,
+  type TypographyProps,
+  Link,
+  Skeleton,
+} from "@mui/material"
+import { BigNumber, ethers } from "ethers"
+import { useMemo } from "react"
+import { ReactComponent as SafeIcon } from "src/assets/images/safe-token.svg"
+import { SelectedDelegate } from "src/components/steps/Claim/SelectedDelegate"
+import { CLAIMING_APP_URL } from "src/config/constants"
+import { useAirdropFile } from "src/hooks/useAirdropFile"
+import { useDelegate } from "src/hooks/useDelegate"
+import { useDelegatesFile } from "src/hooks/useDelegatesFile"
+import { useFetchVestingStatus } from "src/hooks/useFetchVestingStatus"
+import { useTokenBalance } from "src/hooks/useTokenBalance"
+import { sameAddress } from "src/utils/addresses"
+import { formatAmount } from "src/utils/format"
+
+const SpaceContent = styled("div")`
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+`
+
+const Title = (props: TypographyProps) => (
+  <Typography
+    color="primary.main"
+    style={{ fontWeight: "bold", textAlign: "center" }}
+  >
+    {props.children}
+  </Typography>
+)
+
+const Subtitle = (props: TypographyProps) => (
+  <Typography
+    variant="subtitle2"
+    color="primary.light"
+    style={{ marginBottom: "16px", textAlign: "center" }}
+  >
+    {props.children}
+  </Typography>
+)
+
+const StyledButton = (props: ButtonProps) => (
+  <Button
+    size="large"
+    onClick={props.onClick}
+    variant="contained"
+    disableElevation
+    fullWidth
+    disabled={props.disabled}
+  >
+    {props.children}
+  </Button>
+)
+
+const WIDGET_HEIGHT = 300
+
+const ClaimingApp = () => {
```

Comment:
Lets rename the file too.

But lets probably go with `ClaimingWidget`  as I would expect the "main App" as Claiming App